app/services/cleaners/advocate_supplementary_claim_cleaner.rb

```diff
@@ -1,20 +1,8 @@
-module Claim
-  class AdvocateSupplementaryClaimCleaner
-    attr_accessor :claim
-
-    delegate_missing_to :claim
-
-    def initialize(claim)
-      @claim = claim
-    end
-
-    def call
-      destroy_ineligible_misc_fees
-    end
-
+module Cleaners
+  class AdvocateSupplementaryClaimCleaner < BaseClaimCleaner
     private
 
-    def destroy_ineligible_misc_fees
+    def destroy_invalid_fees
       misc_fees.delete(ineligible_misc_fees)
```

Comment:
🤔  not you and not for now but the `.delete` has  the potential to introduce a bug as hooks won't be called. I don't think there are any impacts to worry about for here as the only hook i can find sets the amount on the fee (which is being deleted)